test/runtests.jl

```diff
@@ -24,17 +25,17 @@ eltypes = (Int64, Float64, BigFloat)
             if size(M, 1) == size(M, 2)
                 @test todense(inv(S)) ≈ inv(todense(S))
             end
-            @test show(S) == nothing
-            @test show(stdout, MIME("text/plain"), S) == nothing
+            @suppress_out @test isnothing(show(S))
+            @suppress_out @test isnothing(show(stdout, MIME("text/plain"), S))
```

Comment:
(i) I found it a bit annoying to always have the prints, so I suppressed it (which, unfortunately, introduces an additional dependency...)
(ii) I'll be honest, VSCode complained about `== nothing`, so then I googled and found out that there is `isnothing()` (https://github.com/JuliaLang/julia/issues/29674). Please let me know if this change is not welcome!

Reply:
True, `== nothing` is discouraged, my bad. And thanks for changing it to `isnothing()` :)